manufacturing/closure_panel/README.md

```diff
@@ -51,16 +35,33 @@ BEFORE purchasing any parts.**
 [a5mcmc]: https://www.mcmaster.com/92095A189/
 [a6mcmc]: https://www.mcmaster.com/93475A230/
 [a7mcmc]: https://www.mcmaster.com/93625A150/
+------
+## Custom parts
+
+
+### Closure Panel Fabrication
+The closure panel is made out of a folded, 1.52mm thick, stainless steel sheet.
 
->**#TODO: How much sheet metal?**
 
+ ![](images/fab_drawing_2.jpg)
+
+| Part  | Link |
+|:------|:-------|
+| Closure Panel Fabrication PDF | [.pdf](closure_assembly.pdf)|
+| Closure Panel Fabrication CAD | [.iam](closure_panel.ipt)|
+
+-----
 ## Assembly
 
+ ![](images/fab_drawing_1.jpg)
```

Comment:
So, it's this one that is hard to read in JPG form because the fonts are so small relative to the rest of it.
Also -- the problem I pointed out is still there. The labels appear to be wrong.
* 2 is nowhere to be found
* 1 is pointing at where 2 should be

Reply:
And since they are just 2 pages of the same PDF that you will be linking to, maybe just put them in a row? There isn't so much content here.

Reply:
Im not sure what you're talking about with drawings 1 and 2. Fab 1 is the assembly dwg, 2 is the fab

Reply:
> And since they are just 2 pages of the same PDF that you will be linking to, maybe just put them in a row? There isn't so much content here.

This will further hurt readibility with the small font. Leaving as-is